homeassistant/components/utility_meter/sensor.py

```diff
@@ -90,9 +90,9 @@ def __init__(self, source_entity, name, meter_type, meter_offset=0,
     @callback
     def async_reading(self, entity, old_state, new_state):
         """Handle the sensor state changes."""
-        if any([old_state is None,
-                old_state.state in [STATE_UNKNOWN, STATE_UNAVAILABLE],
-                new_state.state in [STATE_UNKNOWN, STATE_UNAVAILABLE]]):
+        if any([old_state is None, new_state is None]) or\
```

Comment:
It will check it all because you were first generating a list with all the results, then checking. This will not happen if you remove all uses of `any`, as this is not the right place for an any.

Reply:
Your absolutely correct, I was just trying to make the code more readable (hate very long lists of `or`)

(replaced all `any()`)